Mid.css

```diff
@@ -0,0 +1,214 @@
+body
+{ 
+      background-color: white;
+}
+.resume
+{     
+    margin:0 auto;
+
+  width: 1024px;
+  height:1050px;
+
+}	
+.externalDiv
+{
+    height: 200px;
+    width:100%;
+    background-color:RGB(185,200,199);
+}
+.internalDiv
+{
+    float: left;
+    border: 2px solid black;
+    outline: white solid 10px;
+    margin-top: 50px;
+    margin-left:25%;
+    height: 100px;
+    width:50%;
+    background-color: white;
+  padding:1px;
+}
+h2 {
+    text-align: center;
+}
+#header {
+  
+  text-align:center;
+  margin-top:10px;
+  margin-bottom:0px;
+  font-size:30px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+#subHeader {
+  
+   margin-top:3px;
+   text-align:center;
+  font-size:25px;
+   font-style: italic;
+
+}
+.container div
+{
+    
+    display: inline-block;
+    margin:0 auto;
+}
+.container
+{
+    height: 250px;
+    width:auto;
+}
+#para
+{
+    font-family: serif;
+    
+}
+.Left
+{
+  
+
+ padding: 30px;
+}
+.mid
+{
+    
+  
+    position:relative;
+    top:0px;
+    left:70px;
+}
+.Right
```

Comment:
class name should be in small case and if there is multiple words keep a - as a separator

Reply:
Done.